nixos/modules/services/desktops/pipewire/pipewire-media-session.nix

```diff
@@ -37,7 +37,8 @@ in {
     services.pipewire.media-session = {
       enable = mkOption {
         type = types.bool;
-        default = true;
+        default = config.pipewire.enable;
```

Comment:
Shouldn't it be `config.services.pipewire.enable`?

Reply:
Yeah. I messed up cherry-picking. I tested with a different commit.